packages/pluggable-widgets-tools/src/typings-generator/helpers.ts

```diff
@@ -17,3 +17,12 @@ function extractProps<P>(props: Properties, extractor: (props: Properties) => P[
 export function capitalizeFirstLetter(text: string): string {
     return text.charAt(0).toUpperCase() + text.slice(1);
 }
+
+export function commasAnd(arr: any[]) {
+    if (arr.length === 1) {
+        return arr[0];
+    }
+    const firsts = arr.slice(0, arr.length - 1);
+    const last = arr[arr.length - 1];
+    return firsts.join(", ") + " and " + last;
+}
```

Comment:
```suggestion
    return arr.slice(0, -1).join(", ") + (arr.length > 1 ? " and " : "") + arr[arr.length - 1];
}
```
If we're feeling adventurous ;).

Reply:
Applied 😎